eng/pipelines/runtime-community.yml

```
@@ -63,7 +63,7 @@ extends:
           jobParameters:
             testGroup: innerloop
             nameSuffix: AllSubsets_Mono
-            buildArgs: -s mono+libs+host+packs+libs.tests -c $(_BuildConfig) /p:ArchiveTests=true
+            buildArgs: -s mono+libs+host+packs+libs.tests+clr.tools -c $(_BuildConfig) /p:ArchiveTests=true
```

It was actually these two:
```suggestion
            buildArgs: -s mono+libs+host+packs+libs.tests+clr.iltools+clr.packages -c $(_BuildConfig) /p:ArchiveTests=true
```
Please also add them to s390x pipeline few lines above, and trigger the runtime-community pipeline against the PR (by commenting `/azp run runtime-community`).